components/project-card/project-card-detailed.jsx

```diff
@@ -91,19 +92,54 @@ class DetailedProjectCard extends React.Component {
     });
   }
 
+  setSocialPanelHeight() {
+    let article = this.article;
+    let socialIcons = this.socialIcons;
+    let breakpoint = window.matchMedia("(min-width: 992px)");
+
+    function renderHeight(breakpoint) {
+      if (breakpoint.matches) {
+        socialIcons.style.height = `calc(${article.offsetHeight}px)`;
+        article.style.marginTop = `calc(0px - (${
+          article.offsetHeight
+          }px + 80px))`;
+      } else {
+        socialIcons.style.height = `auto`;
+        article.style.marginTop = `auto`;
+      }
+    }
+
+    renderHeight(breakpoint);
+    breakpoint.addListener(renderHeight);
+  }
+
   renderVisitButton() {
     if (!this.props.contentUrl) return null;
 
     return (
-      <a
-        href={this.props.contentUrl}
-        target="_blank"
-        className="btn btn-block btn-info btn-visit text-capitalize mt-3"
-        onClick={() => this.handleVisitBtnClick()}
-      >
-        Visit
-      </a>
-    );
+			<a
```

Comment:
Indentation here is weird. Make sure you're using spaces instead of tabs.

Reply:
Agreed, I have no clue what happened here 🙈